opensrp-chw/src/kk/resources/pnc_hv_hiv_exposed_infant_sw.properties

```diff
@@ -0,0 +1,3 @@
+pnc_hv_hiv_exposed_infant.step1.title = 15. Ufuatiliaji wa mtoto aliyezaliwa na mama mwenye VVU
```

Comment:
Do we need the number in the title?

Reply:
There was a comment from Grace asking we include the numbers as well